### DISCUSSION TITLE
Time To Have benefit student

### DISCUSSION THREAD

**Discussion by Wayvi2** [community/community#132901] [GitHub Education]
### Hi everyone,

Hello, I would like to know if it is normal how long it takes to benefit from the student github pack?

And I must do "Learn new skills" to unlock that ?

Thanks in advance!

**Comment by SamuelOUS** [comment]
"Hi Wayvi2!

The process usually takes no longer than 5 days. If your petition has been accepted, it should be given to you in approximately a week. And no, there's no need to learn new skills to unlock the benefit. Don't worry about it :)"

**Reply by Wayvi2** [reply]
Okay thanks you for replying, I'll wait!

**Reply by SamuelOUS** [reply]
Don't worry, if you want, you can mark this as an answer. It helps a lot!
